run-pipeline-on-rhoai-.md

```diff
@@ -0,0 +1,730 @@
+        path: <prefix-path-to-mixtral-model-in-s3>
+    tolerations:
+    - effect: NoSchedule
+      key: nvidia.com/gpu
+      operator: Exists
+```
+
+</details>
+
+```bash
+oc -n <data-science-project-name/namespace> apply -f servingruntime.mixtral.yaml
+oc -n <data-science-project-name/namespace> apply -f inferenceservice.mixtral.yaml
+```
+
+A new pod named `mixtral-predictor-0000#-deployment-<hash>` should be created. This should result in a successful
+running pod. If the pod does not come up successfully, you inspect the `.status` field for the `InferenceService`
+for issues.
+
+```bash
+oc -n <data-science-project-name/namespace> get inferenceservice mixtral -o yaml
+```
+
+#### Deploy teacher model serving details
+
+Create a secret containing the Teacher model serving details
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <teacher-model-details-k8s-secret>
+  namespace: <data-science-project-name/namespace>
+type: Opaque
+stringData:
+  api_key:  <teacher-model-api-key>                      # Deployed model-server auth token
+  endpoint: <teacher-model-endpoint>                     # Model serving endpoint, Sample format - `https://<deployed-model-server-endpoint>/v1`
+  model: <teacher-model-name>         # Name of the teacher model or deployment
+  SDG_CA_CERT:  <teacher-model-ca-config-map-name>       # Configmap containing CA cert for the teacher model (optional - required if using custom CA cert), Example - `kube-root-ca.crt`
+  SDG_CA_CERT_CM_KEY: <teacher-model-ca-config-map-key>  # Name of key inside configmap (optional - required if using custom CA cert), Example - `ca.crt`
+```
+
+> [!NOTE]
+> Note: If using a custom CA certificate you must provide the relevant data in a ConfigMap. The config map name and
+> key are then provided as a parameter to the pipeline as well as in the `teacher-model-details-k8s-secret` secret above.
+
+If you deployed the Teacher server model using the optional instructions above then you can retrieve `api_key` by
+running the following command:
+
+```bash
+SDG_API_KEY=$(oc -n <data-science-project-name/namespace> create token mixtral-sa)
+```
+
+### (Optional) - Setup NFS StorageClass
+
+> [!CAUTION]
+> The image provided here is for test purposes only.
+> Users must provide a production ready storageclass with ReadWriteMany capability.
+
+This step is needed when the cluster doesn't have a storage provisioner capable of provisioning PersistentVolumeClaim with ReadWriteMany capability.
+
+Installing the NFS CSI driver
+```bash
+$ curl -skSL https://raw.githubusercontent.com/kubernetes-csi/csi-driver-nfs/v4.9.0/deploy/install-driver.sh | bash -s v4.9.0 --`
+```
+
+For deploying an in-cluster NFS server
+
+```bash
+oc new-project nfs
+oc apply -f ./nfs-server-deployment.yaml
+```
+
+> [!NOTE]
+> Note:  Check the root PersistentVolumeclaim that'll be created and the requested storage.
+
+For creating NFS storage-class
+```bash
+oc apply -f ./nfs-storage-class.yaml
```

Comment:
The document doesn't explain where is this file downloaded or what should be the content

Reply:
I see now that the file is in the standalone folder. Consider adding a link in the markdown doc